app/assets/javascripts/components/footer.es6.jsx

```diff
@@ -8,7 +8,7 @@ class Footer extends React.Component {
           <span>    ·    </span>
           <a className="link" href="#">Terms & Privacy</a>
           <span>    ·    </span>
-          <a className="link" href="#">Contact Us</a>
+          <a className="link" href="/contact">Contact Us</a>
```

Comment:
I think it may be wiser in the long run to establish a set of javascript constants (since you'll likely have some constants that can't use rails helpers). Within the javascript constants you can use rails helpers if you'd like. If not a set of javascript constants, definitely do what @vivekraghuram suggested.